### PR TITLE
[14.0][ADD] account_operating_unit_access_all

### PR DESCRIPTION
**Pull request by kittiu**

### CODE REVIEW

File: account_operating_unit_access_all/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Access all OUs' Accounting",
+    "version": "14.0.1.0.0",
+    "author": "Ecosoft,Odoo Community Association (OCA)",
+    "category": "Purchase Management",
```
Comment (newtratip) [inline account_operating_unit_access_all/__manifest__.py]:
```suggestion
    "category": "Accounting & Finance",
```

**Review by AaronHForgeFlow — COMMENTED** [review]
LGTM, after applying the suggested change

**Review by Saran440 — APPROVED** [review]
LGTM :+1:

**Review by ps-tubtim — APPROVED** [review]
LGTM

**Pull request merged**